sdk/core/azure-core/src/samples/java/com/azure/core/http/HttpPipelineBuilderJavaDocCodeSnippets.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.http;
```

Comment:
How about moving the codesnippets into a package like `com.azure.core.docs.*`? This will keep the samples clean for the user to find runnable code samples.

Reply:
This sounds like a good location, let's do this in another PR.

Should we make this a standard convention? Ex. in AppConfiguration they'll use `com.azure.data.appconfiguration.docs.*`

Reply:
Yeah, this should be a standard convention for all client libraries.